README.md

```diff
@@ -23,8 +23,10 @@ Globally install azure function tools:
 In one terminal run the next development server:
 
 ```bash
-nvm use && yarn dev
+nvm use 14.0.0 && yarn dev
```

Comment:
This worked for me, but we have two `.nvmrc` files with different node versions. This one seems to be what we run in the root folder, but that has a `.nvmrc` file with node `v16` in it.

I was wondering if we should keep the original command but then add a subheading for what to do if you have issues running this command with an M1 MacBook.

Reply:
Thanks for looking @Renny4Real ! It makes an absolute sense to create a new section for this and leave the original instructions as they are. If that's OK, I'll resolve this conversation and keep the one with Bhavesh opened, because he's said the same thing as you.